nishimweelysee/Elysee-CapstoneProject
# Ft number of entries

#### What does this PR do?
> This PR displays number of entries made since registration.
>
#### Description of Task to be completed
> input field that display number of entries
>
#### How should this be manually tested?
> 1. open website
>2. go to admin page
>3. go to profile Tab
>4. click on number of entry button
> then you will see number of entries made.
#### Any background context you want to provide?
>
#### What are the relevant Trello story?
> Ft(number of Entries made since registration)  use this link to see my card :https://trello.com/c/8KTYv7Ct
<img width="333" alt="num1" src="https://user-images.githubusercontent.com/56052588/92458085-46fc1b80-f1c5-11ea-93db-5efa2b771def.PNG">

>
<img width="304" alt="num2" src="https://user-images.githubusercontent.com/56052588/92458095-49f70c00-f1c5-11ea-9440-7e5ef64f43fa.PNG">

> Thank you!!
#### Screenshots (if appropriate)
>
